sorted-r-blacklist

```diff
@@ -20,15 +20,19 @@
 #recipes/r-fastcluster
 #recipes/r-sendmailr
 #recipes/bioconductor-biocgenerics
-recipes/r-maldiquant
-recipes/r-fda
-recipes/r-compute.es
-recipes/r-snowfall
-recipes/r-permute
-recipes/r-funr
-recipes/r-calibrate
+#recipes/r-maldiquant
+#recipes/r-fda
+#recipes/r-compute.es
+#recipes/r-snowfall
+#recipes/r-permute
+#recipes/r-funr
+#recipes/r-calibrate
+
+# r-ash needs gfortran; even after adding gcc it complains about needing
```

Comment:
I saw this once, but was this a error or a warning? If an error we need to patch the source I guess :(